libs/remix-core-plugin/src/lib/compiler-content-imports.ts

```diff
@@ -88,21 +87,23 @@ export class CompilerImports extends Plugin {
     }
   }
 
-  importExternal (url, targetPath, cb) {
-    this.import(url,
-      // TODO: handle this event
-      (loadingMsg) => { this.emit('message', loadingMsg) },
-      async (error, content, cleanUrl, type, url) => {
-        if (error) return cb(error)
-        try {
-          const provider = await this.call('fileManager', 'getProviderOf', null)
-          const path = targetPath || type + '/' + cleanUrl
-          if (provider) provider.addExternal('.deps/' + path, content, url)
-        } catch (err) {
-
-        }
-        cb(null, content)
-      }, null)
+  importExternal (url, targetPath) {
```

Comment:
A clean approach may be making this function itself `async`

Reply:
same issue as above, `import` rely mostly on a callback. This can certainly be done but not sure we have place for that in this PR.